src/mame/mr/nl_dribling.h

```diff
@@ -0,0 +1,12 @@
+// license:BSD-3-Clause
+// copyright-holders: Enzo Lombardi
+
+#ifndef _NL_DRIBBLING_H
+#define _NL_DRIBBLING_H
```

Comment:
Please use the conventional form for `#include` guards.  Also, any name beginning with an underscore followed by an uppercase letter is reserved in both C and C++.

Reply:
Done